src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/prior_box.cpp

```diff
@@ -105,9 +109,9 @@ class PriorBoxLayerGPUTest : public testing::WithParamInterface<PriorBoxLayerGPU
                 attributes_clustered.widths = {86, 13, 57, 39, 68, 34, 142, 50, 23};
                 attributes_clustered.heights = {44, 10, 30, 19, 94, 32, 61, 53, 17};
                 attributes_clustered.variances = {0.1, 0.1, 0.2, 0.2};
-                attributes_clustered.step = 16;
-                attributes_clustered.step_widths = 0;
-                attributes_clustered.step_heights = 0;
+                attributes_clustered.step = 1;
```

Comment:
why are these values changed?

Reply:
It's not important. But I rollbacked to the original value. Thanks!